### PR TITLE
add alphas variations NNLO and NLO

### PR DESCRIPTION
**Pull request by RoyStegeman**
Extend the range of alphas theories at NNLO 

Add the corresponding NLO theories

### CODE REVIEW

File: nnpdf_data/nnpdf_data/theory_cards/837.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+ID: 837
+PTO: 2
+FNS: FONLL-B
```
Comment (andreab1997) [inline nnpdf_data/nnpdf_data/theory_cards/837.yaml:3]: I believe that this theory card for NLO should change a but, for example it is missing the DAMPOWERb and DAMPOWERc entries

**Review by andreab1997 — CHANGES_REQUESTED** [review]

File: nnpdf_data/nnpdf_data/theory_cards/837.yaml (hunk 1 above)
Comment (RoyStegeman) [reply nnpdf_data/nnpdf_data/theory_cards/837.yaml:3]: PTO is wrong :sweat_smile: 

But DAMPPOWER are not present in the nnpdf theorycards. They will only be after #2028

Comment (andreab1997) [reply nnpdf_data/nnpdf_data/theory_cards/837.yaml:3]: Yes indeed, I agree with that.

Sorry that I have not noticed the PTO 😂

**Comment by RoyStegeman** [comment]
I should probably revert deleting the values for which there are defaults. Seems that #2028 is not stable yet

**Comment by RoyStegeman** [comment]
@andreab1997 what should we do with these? I was waiting for the single truth, but maybe we should just revert to the old format, so same as all the others, and merge?

**Pull request merged**